spectree/plugins/falcon_plugin.py

```diff
@@ -223,7 +224,12 @@ def validate(
             return
 
         func(*args, **kwargs)
-        if resp and resp.has_model():
+
+        if isinstance(_resp.media, BaseModel):
+            _resp.media = _resp.media.dict()
+            skip_validation = True
```

Comment:
I guess it comes down to how integrated you want this functionality to be?

For my use case we work with pydantic everywhere so we don't need spectree to validate anything, pydantic already does it, as a workaround we return `model.dict(by_alias=True)` from our views so spectree is happy.

We also handle setting the response status code ourselves for errors so we don't need spectree to actually check the model we return.

Though I can see it being handy for spectree to check the model type against `resp.models()` and still validating the model is at least a model the view should return - it could then also automatically set the status code from the model type.

Reply:
I see. I think it's better to align with the current logic. In this way, the response content is verified by `pydantic`, the content type is verified by `spectree` to make sure it's the same as defined in the generated OpenAPI doc.


```suggestion
        if isinstance(_resp.media, resp.find_model(_resp.status[:3])):
            _resp.media = _resp.media.dict()
            skip_validation = True
```

Reply:
Thanks, I've implemented this.